src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
```

Comment:
<img width="652" alt="Знімок екрана 2023-09-26 о 18 40 07" src="https://github.com/mate-academy/jv-lottery/assets/41166578/01cfdc2e-ee9d-4bd0-a15a-616eec78fdf7">

```suggestion
    public Ball getRandomBall() {
```